acceptance/suites/tests/010-puppetserver-cli/subcommand/SERVER-297_common_behavior.rb

```diff
@@ -13,6 +15,18 @@
   assert(env['JARS_NO_REQUIRE'], "JARS_NO_REQUIRE missing")
 end
 
+step "ruby: Check that proxy env-variables are present"
+on(master, "#{proxy_env_vars} puppetserver ruby -rjson -e 'puts JSON.pretty_generate(ENV.to_hash)'") do
+  env = JSON.parse(stdout)
+  assert(env['HTTP_PROXY'], "HTTP_PROXY is missing")
```

Comment:
I think it would be good to have each of these assertions validate that the actual value of the environment variable was passed through properly, e.g., that `env['HTTP_PROXY']` is equal to "foo".  WDYT?